Patch to fix issue with GS & HP Printers
Bug # 15777965.

NOTE : This patch will not be pushed upstream.

--- ghostscript-9.25/cups/gdevcups.c	2018-10-17 01:45:12.760567053 +0000
+++ ghostscript-9.25/cups/gdevcups.c	2018-10-17 01:42:11.177584155 +0000
@@ -3880,7 +3880,7 @@ cups_put_params(gx_device     *pdev,	/*
 
       if ((code = gdev_prn_maybe_realloc_memory((gx_device_printer *)pdev,
                                                 &sp_old,
-						width_old, height_old,
+						width, height,
 						transp_old))
 	  < 0)
         goto done;
